doc/doxygen/headers/concepts.h

```diff
@@ -49,6 +49,27 @@
  * throughout deal.II. Specifically, in the example above, the
  * @ref ConceptNumber "Number concept" discussed below describes the
  * types that could be used as argument for the Vector class.
+ *
+ * (As a side note, concepts have been proposed as a language
```

Comment:
I looked at the rendered copy of this and this side note is nearly as long as the rest of the preamble. I think it would make since to delete "(As a side note," and just start with "Concepts have been proposed..."